src/services/panel.ts

```diff
@@ -31,24 +31,35 @@ import { getLabelsFromSeries, getVisibleFields, getVisibleLabels, getVisibleMeta
 import { getParserForField } from './fields';
 
 const UNKNOWN_LEVEL_LOGS = 'logs';
+export const INFO_LEVEL_FIELD_NAME_REGEX = /^info$/i;
+export const DEBUG_LEVEL_FIELD_NAME_REGEX = /^debug$/i;
+export const WARNING_LEVEL_FIELD_NAME_REGEX = /^(warn|warning)$/i;
+export const ERROR_LEVEL_FIELD_NAME_REGEX = /^error$/i;
+export const CRITICAL_LEVEL_FIELD_NAME_REGEX = /^(crit|critical|fatal)$/i;
+export const UNKNOWN_LEVEL_FIELD_NAME_REGEX = /^(logs|unknown)$/i;
+
 export function setLevelColorOverrides(overrides: FieldConfigOverridesBuilder<FieldConfig>) {
-  overrides.matchFieldsWithNameByRegex('/^info$/i').overrideColor({
+  overrides.matchFieldsWithNameByRegex(new RegExp(INFO_LEVEL_FIELD_NAME_REGEX).source).overrideColor({
```

Comment:
```suggestion
  overrides.matchFieldsWithNameByRegex(INFO_LEVEL_FIELD_NAME_REGEX.source).overrideColor({
```
Should do it, right?